libra_swarm/src/main.rs

```diff
@@ -45,37 +47,81 @@ fn main() {
         faucet_key_file_path
     );
 
-    let swarm = LibraSwarm::launch_swarm(
+    let mut validator_swarm = LibraSwarm::launch_swarm(
         num_nodes,
+        RoleType::Validator,
         !args.enable_logging,
-        faucet_account_keypair,
+        faucet_account_keypair.clone(),
         args.config_dir.clone(),
         None, /* template_path */
+        None, /* upstream_config_dir */
     );
+    let full_node_swarm = if args.with_full_nodes {
+        let swarm = LibraSwarm::launch_swarm(
+            num_nodes,
+            RoleType::FullNode,
+            !args.enable_logging,
+            faucet_account_keypair,
+            None, /* config dir */
+            None, /* template_path */
+            Some(String::from(
+                validator_swarm
+                    .dir
+                    .as_ref()
+                    .expect("Validator swarm config directory not set")
+                    .as_ref()
+                    .join("0")
+                    .to_str()
+                    .expect("Failed to convert std::fs::Path to String"),
+            )),
+        );
+        validator_swarm.kill_node(0);
```

Comment:
maybe in follow up diff, but can we change API a bit to avoid node restart(feels a bit hacky)?
So launch swarm will be actually 2 step process: configure swarm and then start it
This way we can configure validator swarm first, then configure full node swarm, and only then start validator swarm

Reply:
yeah, sounds better